theme_vehicle/views/new_page_template.xml

```diff
@@ -115,10 +115,17 @@
     </xpath>
 </template>
 
+<template id="new_page_template_s_cover" inherit_id="website.new_page_template_s_cover">
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" remove="o_half_screen_height" separator=" "/>
+    </xpath>
+</template>
+
+
 <template id="new_page_template_landing_0_s_cover" inherit_id="website.new_page_template_landing_0_s_cover">
     <xpath expr="//section" position="attributes">
         <!-- pb256 is defined in both theme and new page template -->
-        <attribute name="class" add="pb256" remove="pt0 pb256" separator=" "/>
+        <attribute name="class" add="pb80 pt80" remove="pt256 pb256 pt0 pb0" separator=" "/>
```

Comment:
After theme customization, the default `s_cover` will have `pt0 pb0`. Why is the `pt256 pb256` removal needed here :thinking: ?

Reply:
Without that, a new page template (landing cover 0) wasn't functionning properly and test was failing because of duplicate class. I remember spending some time trying debugging at that time, and that worked... : / If you have any other ideas..